src/stories/Table.stories.tsx

```diff
@@ -215,7 +215,15 @@ export const TxTable: Story = {
             }
             columns={getStorybookTxTableTanstackColumns(args.hasSorting, args.pinFirstColumn)}
             data={args.isEmpty ? [] : storybookTxTableRowData}
-            bannerRow={args.bannerRow ? <UpdateTableBannerRow onClick={() => {}} /> : undefined}
```

Comment:
Can we add a story for the new blocks table, as I don't know where I can visually test the new block table